client.go

```diff
@@ -135,6 +135,22 @@ func New(address string, opts ...ClientOpt) (*Client, error) {
 	if copts.services == nil && c.conn == nil {
 		return nil, errors.New("no grpc connection or services is available")
 	}
+
+	// check namespace labels for default runtime
+	defaultns := "default"
+	if copts.defaultns != "" {
+		defaultns = copts.defaultns
+	}
+	namespaces := c.NamespaceService()
+	ctx := context.Background()
+	if labels, err := namespaces.Labels(ctx, defaultns); err == nil {
```

Comment:
This should probably only be looked up when a defaultns is provided and no defaultruntime is. The given option should always be preferred and I don't think we should default the ns here.

Also I think `NewWithConn` should probably have matching logic